card/mod/machines/set/all/reset_machines.rb

```diff
@@ -1,12 +1,11 @@
 module ClassMethods
   def reset_all_machines
     Auth.as_bot do
-      Card.search(
-        right: { codename: %w[in machine_cache machine_output] }
-      ).each do |card|
+      Card.search(right: { codename: "machine_output" }).each do |card|
```

Comment:
not that it matters, but I think search takes a block, so the "each" isn't necessary.